Cargo.toml

```diff
@@ -17,6 +17,7 @@ license     = "Apache-2.0/MIT"
 exclude     = ["book/*"]
 
 [dependencies]
+anes           = "0.1"
```

Comment:
I tested this PR with `-Z minimal-versions` and noticed that it won't compile due to the version of `anes` being a minimum of 0.1. With the version set to 0.1.4 it compiles properly.